cqrs-user-query-service/src/main/java/dev/amir/userquery/framework/output/adapter/UserOutputAdapter.java

```diff
@@ -27,11 +30,13 @@ public Collection<User> getAll() {
 
     @Override
     public Optional<User> getById(String userId) {
+        log.info("Getting user by ID: {} from the database.");
```

Comment:
same here, no parameter to be printed

Reply:
log.info("Getting user by ID: {} from the database.", userId);